fiftyone/core/collections.py

```diff
@@ -9044,6 +9045,36 @@ def values(
         Returns:
             the list of values
         """
+
+        # If we do not need to follow insertion order, we can potentially use a
+        # covered index query to get the values directly from the index and
+        # avoid a COLLSCAN.
+        if not _enforce_natural_order:
+            field = None
+            if isinstance(field_or_expr, str):
+                field = field_or_expr
+            elif isinstance(field_or_expr, list):
+                field = field_or_expr[0]
```

Comment:
this will return the wrong values if you called `ds.values(["id", "filepath"])` - it would return the same as `ds.values('id')`.

I suggest removing this and we will note it for follow on work.

```suggestion
```

Reply:
Ok!